runtimes/pythonrt/dev-requirements.txt

```diff
@@ -2,3 +2,12 @@
 
 pytest ~= 8.0
 ruff ~= 0.2
+
+# AutoKitteh integrations
+
+atlassian-python-api ~= 3.41
+google-api-python-client ~= 2.122
+google-auth-httplib2 ~= 0.2
+google-auth-oauthlib ~= 1.2
+jira ~= 3.8
+slack-sdk ~= 3.27
```

Comment:
These should be in `requirements.txt` since `ak` uses it to build the virtual environment.
Maybe you can update the comment at the top of `requirements.txt` to be clearer?